### PR TITLE
Remove metadata model

### PR DESCRIPTION
**Pull request by DanielElisenberg**
# REMOVE METADATA MODEL

Cleaning up repo before adding new changes to validation.

### What's changed?
* Removed metadata model after it has been moved to job-executor
* General cleanup and linting
* Bumped version to 3.1.1 => 4.0.0

### CODE REVIEW

File: microdata_validator/dataset_reader.py [hunk 1]
```diff
@@ -91,7 +92,7 @@ def _read_and_process_data(data_file_path: Path,
                     value = data_row[1].strip('"')
                     start = data_row[2].strip('"')
                     stop = data_row[3].strip('"')
-                    # TODO: attributes = data_row[4]
+                    # attributes = data_row[4]
```
Comment (pawbu) [inline microdata_validator/dataset_reader.py:95]: Why is it commented out? I would like a description here

File: microdata_validator/dataset_reader.py [hunk 2]
```diff
@@ -128,30 +129,32 @@ def _read_and_process_data(data_file_path: Path,
                                 f"row {reader.line_num}: "
                                 f"STOP date not valid - '{start}'"
                             )
-                    # TODO: validate "attributes"?
+                    # validate "attributes" here
```
Comment (pawbu) [inline microdata_validator/dataset_reader.py:132]: I think this should have the TODO tag otherwise could be removed

**Review by pawbu — APPROVED** [review]
Looks good, some minor comments

File: microdata_validator/dataset_reader.py (hunk 1 above)
Comment (DanielElisenberg) [reply microdata_validator/dataset_reader.py:95]: It is not really on the plan right now to validate additional attributes, so it felt weird to have a TODO for something we might not do. I could remove these comments to like you mention below? That seems fair 👍🏻

Comment (pawbu) [reply microdata_validator/dataset_reader.py:95]: Right, removing them will make the intent clearer I think 👍

**Pull request merged**